### DISCUSSION TITLE
Enable "playground" for completed lessons?

### DISCUSSION THREAD

**Discussion by guppy42** [GDQuest/learn-gdscript#111] [Ideas]
Currently you are immediately advanced to the next lesson when you provide the correct solution  - I think it would be better to replace the "got it" button with;

- 'playground/tinker mode' - returns you to the current lesson where you can have a play with the code to see what altering it would do ( but still have the option to go to the next lesson with out having to change the code back to the correct solution )
- 'next lesson' - the current "got it" functionality

Due to #58 you cant just go back to the lesson - but even with that fixed there would be no easy/obvious way to resume the lessons (with out having to first provide the correct solution)

My 11yo son got so frustrated with not being able to tinker with the code that he ended up copying the code to a new GD project - which of course doesn't work as the "turtle functions" aren't available outside the tutorial.

**Comment by YuriSizov** [comment]
Yes, there are a couple of reported issues that discuss this already. I plan to work on the end of the practice and end of the lesson experience next week to address those concerns. You will be able to stay on the page and play more with it.
